prometheus/handler.go

```diff
@@ -47,6 +48,16 @@ type Handler struct {
 
 	opcount uint64
 	metrics metricStore
+	filters []byte
+}
+
+// FilterLabels will remove the passed labels from the observation
+func (h *Handler) FilterLabels(labelNames []string) {
```

Comment:
By a function named `filter` I would expect it to keep only the matching values instead of filtering them out. Shouldn't we use a different name in this case?

Reply:
Some examples:
* `IgnoreLabels`
* `ExcludeLables`
* `OmitLables`

Reply:
went with `IgnoreLabels`